src/core/connection.h

```diff
@@ -312,11 +312,7 @@ typedef struct QUIC_CONN_STATS {
 //
 typedef struct QUIC_CONNECTION {
 
-#ifdef __cplusplus
-    struct QUIC_HANDLE _;
-#else
-    struct QUIC_HANDLE;
-#endif
+    struct QUIC_HANDLE Self;
```

Comment:
You can't rename this for the C code because it will break everywhere. If you want, you can copy what we do with `QUIC_CONNECTION` to the other structs, but don't change this.

Reply:
uh oh